core/trino-main/src/test/java/io/trino/operator/BenchmarkGroupByHash.java

```diff
@@ -119,7 +148,13 @@ public Object addPagePreCompute(BenchmarkData data)
     public Object bigintGroupByHash(SingleChannelBenchmarkData data)
     {
         GroupByHash groupByHash = new BigintGroupByHash(0, data.getHashEnabled(), EXPECTED_SIZE, NOOP);
-        data.getPages().forEach(p -> groupByHash.addPage(p).process());
+        for (Page page : data.getPages()) {
```

Comment:
nit: we can extract this in a method

Reply:
Maybe, in general- helper methods are a little tricky for benchmarks because they may or may not inline even code around them changes and the differences can skew measurements.

Reply:
> helper methods are a little tricky for benchmarks because they may or may not inline even code around them changes and the differences can skew measurements.

IMO that doesn't matter since this helper methods processes pages, not rows. We don't really care all that much about page processing overhead

Reply:
Ok, added a helper method.